articles/api/management/v2/user-search/v3/index.md

```diff
@@ -0,0 +1,304 @@
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "q",
+          "value": "email:\"jane@exampleco.com\""
+        },
+        {
+          "name": "search_engine",
+          "value": "v3"
+        }
+    ]
+}
+```
+
+::: note
+For more information on other available parameters, check out the [Management API Explorer documentation](/api/management/v2#!/Users/get_users).
+:::
+
+## Query string syntax
+
+When searching for users, you can create custom queries using Lucene Query Syntax.
+
+The query string is parsed into a series of terms and operators:
+
+* A term can be a single word such as `jane` or `smith`.
+* A can be a phrase surrounded by double quotes (`"jane smith"`) which will match all the words in the phrase in the same order.
+* A term without a field name will not match text in the `app_metadata`/`user_metadata` fields.
+* Multiple terms can be grouped together with parentheses to form sub-queries.
+* Search values are case insensitive.
+* Operators (`AND`, `OR`, `NOT`) work on all normalized user fields and root metadata fields.
+
+### Searchable fields
+
+You can search for users using the following fields:
+
+* All the [normalized user profile](/user-profile/normalized/auth0) fields.
+* `app_metadata` and `user_metadata` fields may be used with:
+    * Booleans
+    * Numeric (integer or double)
+    * Text
+    * Arrays. You can search for strings and numerics in metadata arrays on the root object, but not in nested fields.
+
+### Wildcards
+
+::: note
+Search using wildcards is not available on `app_metadata`/`user_metadata` fields.
+:::
+
+Wildcard searches can be run on terms using the asterisk character (`*`) to replace zero or more characters: `name:john*`. The question mark character (`?`), is currently not supported.
+
+For example, to find all users whose names start with "john", use `q=name:john*`:
+
+```har
+{
+    "method": "GET",
+    "url": "https://${account.namespace}/api/v2/users",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "q",
+          "value": "name:john*"
+        },
+        {
+          "name": "search_engine",
+          "value": "v3"
+        }
+    ]
+}
+```
+
+### Exact match
+
+To find exact matches, use double quotes: `name:"jane smith"`.
+
+For example, to find users with name start with "jane smith", use `q=name:"jane smith"`:
+
+```har
+{
+    "method": "GET",
+    "url": "https://${account.namespace}/api/v2/users",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "q",
+          "value": "name:\"jane smith\""
+        },
+        {
+          "name": "search_engine",
```

Comment:
Is this a required request param? What if I don't set this? Does it use v2 by default?